### PR TITLE
Counterspell Calgary

### PR DESCRIPTION
**Pull request by NoozAbooz**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/calgary**](https://counterspell-9989dgjbm.hackclub.dev/calgary) |
|:-:|
|<a href="https://counterspell-9989dgjbm.hackclub.dev/calgary">Screenshot of /calgary</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/IjCbNYTxRPxxibrThsGZtGk1klXaLyzSmNbRPLTXPpS5cktNjcvOXsNXl2mdXSmkLwTre7cwZlbMKMWePb2RFg.png">(view full size)</a> |







Commit 8931b14f1b3a722e1e3f4ad6114f43063933dbed (https://counterspell-9989dgjbm.hackclub.dev).

File: package.json [hunk 1]
```diff
@@ -29,5 +29,6 @@
     "eslint-config-next": "14.2.14",
     "postcss": "^8",
     "tailwindcss": "^3.4.1"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```
Comment (samdev-7) [inline package.json:33]: Please revert this change

```suggestion
  }
```

File: yarn.lock
Comment (samdev-7) [inline yarn.lock:1]: The packages recently updated, you might need to re-pull and re-install with yarn. Alternatively, just revert the changes in this file.

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by NoozAbooz** [comment]
Is this correct?

File: yarn.lock
Comment (samdev-7) [inline yarn.lock:1]: The packages were updated recently, you might need to pull & re-install with yarn. Alternatively, just revert this file.

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by samdev-7** [comment]
It seems to still have issues, I'll revert it for you.

**Review by samdev-7 — APPROVED** [review]
